src/sql/run/run.py

```diff
@@ -33,6 +33,11 @@ def run_statements(conn, sql, config, parameters=None):
         return "Connected: %s" % conn.name
 
     for statement in sqlparse.split(sql):
+        # filter statements that only contain comments
```

Comment:
This comment is not very clear. 
In which case will statement only contain comments? It looks like we are just stripping off all comments from every statement that `sqlparse` returns?

Reply:
Ok I got it now, it will be used to extract the last comment after semicolon. Better to mention this specific use-case  as comments

Reply:
Added [more informative comments](https://github.com/ploomber/jupysql/pull/939/files#diff-86d3c4bd76fcf79123c806782f9298eccedad8167c5f1af692d10d5fca4436bcR36-R41)